about.md

```diff
@@ -4,18 +4,53 @@ title: About
 is_nav: true
 permalink: /about/
 ---
+開発、改善、海賊
 
+Astral Dynamics is an Innovation and Rapid Proto-typing agency. We are a multi-
+disciplinary team that brings genuine creativity together with exception 
+technical knowledge and skill. We respond to challenges, design solutions and rapid prototype.
 
-Astral Dynamics is a forward thinking tech startup with a focus on improving and enhancing culture.
+We follow our own Agile methodology and have a unique organisational culture based 
+on the naval metaphore. Every project is a voyage of discovery. We believe very 
+strongly in our freedom and while we occasionally hire out our servives to worthy
+causes, we don't compromise on our ideals or sell what we hold sacred.
 
-We're normally UK based, with our offices in North Wales, but this year we decided to take our company out on the road in Asia --- read more about our adventure [here][1].
+Our home base is in Snowdonia in North West Wales. An [magical][1] place, home 
```

Comment:
Referencing druids is a surefire way to make sure many people don't take us at all seriously.